src/Symfony/Component/HttpKernel/Tests/Fixtures/TestEventDispatcher.php

```diff
@@ -29,4 +29,9 @@ public function getNotCalledListeners()
     public function reset()
     {
     }
+
+    public function getOrphanedEvents()
```

Comment:
any tests to update? given `TestEventDispatcher extends EventDispatcher implements TraceableEventDispatcherInterface` is deprecated behavior.